2022/Q2/artifacts/pos-model/PoS-model.md

```diff
@@ -134,9 +124,11 @@ tx_unbond(validator_address, amount)
 ```go
 tx_withdraw_unbonds_validator(validator_address)
 {
-  //COMMNET: in the docs, the system panics if no self-unbonds. Why? In our view, if a user
-  //attemps to withdraw but has no self-unbounds, the the transaction is a noop. Is there something
-  //wrong about assuming that?
+  /* COMMENT
+  in the docs, the system panics if no self-unbonds. Why? In our view, if a user
+  attemps to withdraw but has no self-unbounds, the the transaction is a noop. Is there something
+  wrong about assuming that? same for tx_withdraw_unbonds_delegator
+  */
```

Comment:
In line 155 below: s/src_validator_address/validator_address/
(thanks Github for the amazing feature set -_-)